app/views/content_items/case_study.html.erb

```diff
@@ -19,12 +19,9 @@
           alt: @content_item.image["alt_text"],
           caption: @content_item.image["caption"] if @content_item.image %>
 
-      <%= render 'components/important-metadata',
-          items: @content_item.metadata[:other] %>
-
       <%= render 'govuk_component/govspeak',
-        content: @content_item.body,
-        direction: page_text_direction %>
+          content: @content_item.body,
```

Comment:
Is this additional indentation intentional?

Reply:
Yeah, just to match it up with the indentation when rendering the figure component on line 17.  I think it looks neater like this.